### DISCUSSION TITLE
Welcome to SummerSchool-Assignment Discussions!

### DISCUSSION THREAD

**Discussion by Vinayak-VG** [Vinayak-VG/SummerSchool-Assignment#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to solve your doubts regarding the assignment if you have any. We hope that you ask questions you’re wondering about.

**Comment by SysRunAI** [comment]
Regarding question 1, what are yi and yi cap?  Is y random combination of 0's and 1's?

**Reply by aryanpandey** [reply]
yi is a combination of 0s and 1s which yhat is a vector whose elements are any real number between 0 and 1
